charts/runtime/values.yaml

```diff
@@ -80,9 +80,18 @@ ingress-nginx-private:
         networking.gke.io/load-balancer-type: "Internal"
         service.beta.kubernetes.io/azure-load-balancer-internal: "true"
         service.beta.kubernetes.io/aws-load-balancer-scheme: internal
+        service.beta.kubernetes.io/aws-load-balancer-backend-protocol: tcp
+        service.beta.kubernetes.io/aws-load-balancer-cross-zone-load-balancing-enabled: 'true'
+        service.beta.kubernetes.io/aws-load-balancer-type: external
+        service.beta.kubernetes.io/aws-load-balancer-nlb-target-type: ip
+        service.beta.kubernetes.io/aws-load-balancer-proxy-protocol: "*"
+        service.beta.kubernetes.io/aws-load-balancer-connection-idle-timeout: '3600'
     admissionWebhooks:
       enabled: false
     config:
+      compute-full-forwarded-for: 'true'
```

Comment:
these can't be in the chart default values because it will make all clouds enable proxy protocol (move it to the helm values and comment it out w/ a description that it should be enabled on aws)